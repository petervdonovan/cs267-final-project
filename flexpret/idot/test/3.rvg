[mu []
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [get-hartid a0]
    [single-core a1
      [parfor t0 t1 11 to 75 by 1
        [critical-hart t3 t4
          [sim-print t0 t2]]]]
    [id die]}]]
]
