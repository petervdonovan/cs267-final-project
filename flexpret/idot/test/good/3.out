Core-0:         11
Core-0:         12
Core-0:         13
Core-0:         14
Core-0:         15
Core-0:         16
Core-0:         17
Core-0:         18
Core-0:         19
Core-0:         20
Core-0:         21
Core-0:         22
Core-0:         23
Core-0:         24
Core-0:         25
Core-0:         26
Core-0:         27
Core-0:         28
Core-0:         29
Core-0:         30
Core-0:         31
Core-0:         32
Core-0:         33
Core-0:         34
Core-0:         35
Core-0:         36
Core-0:         37
Core-0:         38
Core-0:         39
Core-0:         40
Core-0:         41
Core-0:         42
Core-0:         43
Core-0:         44
Core-0:         45
Core-0:         46
Core-0:         47
Core-0:         48
Core-0:         49
Core-0:         50
Core-0:         51
Core-0:         52
Core-0:         53
Core-0:         54
Core-0:         55
Core-0:         56
Core-0:         57
Core-0:         58
Core-0:         59
Core-0:         60
Core-0:         61
Core-0:         62
Core-0:         63
Core-0:         64
Core-0:         65
Core-0:         66
Core-0:         67
Core-0:         68
Core-0:         69
Core-0:         70
Core-0:         71
Core-0:         72
Core-0:         73
Core-0:         74
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
